jenkins/Jenkinsfile

```diff
@@ -482,53 +497,34 @@ timestamps {
                             }
                         }
 
+                        stage ('Generate Workspace Info') {
+                            withCredentials([string(credentialsId: '92cf81e5-6db5-408c-8a0d-192b36200a16', variable: 'GITHUB_AUTH_TOKEN'), string(credentialsId: 'xamarin-build-tasks', variable: 'XAMARIN_BUILD_TASKS')]) {
```

Comment:
The rest of the file uses space before opening parenthesis (like the mono coding guidelines).